manual_tests/internal/nosend_sendwith.go

```diff
@@ -0,0 +1,172 @@
+package internal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/bsv-blockchain/go-sdk/chainhash"
+	"github.com/bsv-blockchain/go-sdk/transaction"
+	sdk "github.com/bsv-blockchain/go-sdk/wallet"
+	"github.com/bsv-blockchain/go-wallet-toolbox-manual-tests/internal/fixtures"
+	"github.com/bsv-blockchain/go-wallet-toolbox-manual-tests/internal/tui"
+	"github.com/go-softwarelab/common/pkg/to"
+)
+
+type noSendTransactionsResult struct {
+	TxHashes      []chainhash.Hash
+	NoSendTimes   []time.Duration
+	NoSendChange  []transaction.Outpoint
+	TotalTxCount  int
+	MinNoSendTime time.Duration
+	MaxNoSendTime time.Duration
+	AvgNoSendTime time.Duration
+}
+
+type sendWithTransactionResult struct {
+	SendWithTime     time.Duration
+	Success          bool
+	TxHash           *chainhash.Hash
+	BroadcastedTxIds []chainhash.Hash
+}
+
+func (m *Manager) ExecuteNoSendSendWith(user fixtures.UserConfig, txCount int, dataPrefix string) (*tui.NoSendSendWithResult, error) {
+	noSendResult, err := m.executeNoSendTransactions(user, txCount, dataPrefix)
+	if err != nil {
+		return nil, fmt.Errorf("failed to execute noSend transactions: %w", err)
+	}
+
+	sendWithResult, err := m.executeSendWithTransaction(user, noSendResult.TxHashes)
+	if err != nil {
+		return nil, fmt.Errorf("failed to execute sendWith transaction: %w", err)
+	}
+
+	return &tui.NoSendSendWithResult{
+		NoSendTimes:      noSendResult.NoSendTimes,
+		SendWithTime:     sendWithResult.SendWithTime,
+		MinNoSendTime:    noSendResult.MinNoSendTime,
+		MaxNoSendTime:    noSendResult.MaxNoSendTime,
+		AvgNoSendTime:    noSendResult.AvgNoSendTime,
+		TotalTxCount:     noSendResult.TotalTxCount,
+		BroadcastedTxIds: sendWithResult.BroadcastedTxIds,
+	}, nil
+}
+
+func (m *Manager) executeNoSendTransactions(user fixtures.UserConfig, txCount int, dataPrefix string) (*noSendTransactionsResult, error) {
+	userWallet, err := m.WalletForUser(user)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get wallet for user %s: %w", user.Name, err)
+	}
+
+	var noSendTimes []time.Duration
+	var txHashes []chainhash.Hash
+	var noSendChange []transaction.Outpoint
+
+	for i := 0; i < txCount; i++ {
+		startTime := time.Now()
+
+		opReturnData := fmt.Sprintf("%s%d", dataPrefix, time.Now().Unix())
```

Comment:
```suggestion
		opReturnData := fmt.Sprintf("%s_%d", dataPrefix, i)
```

When I was testing it, I realized that the timestamp is not readable. Consider just a counter here

Reply:
changed to `i+1`